src/Hangfire.SqlServer/SqlServerMonitoringApi.cs

```diff
@@ -466,7 +585,24 @@ private long GetNumberOfJobsByStateName(SqlConnection connection, string stateNa
 
             return count;
         }
-
+        private string[] prepareSqlFilterDates([NotNull]string startDate, [NotNull]string endDate)
+        {
+            string start, end;
```

Comment:
@RobertGill @quickskape I prefer it on two lines instead of one line declaration. What do you guys think?